dbusutil/dbustest/dbustest.go

```diff
@@ -34,10 +34,10 @@ const testDBusClientName = ":test"
 // DBusHandlerFunc is the type of handler function for interacting with test DBus.
 //
 // The handler is called for each message that arrives to the bus from the test
-// client. The handler can respond by returning zero or more messages.
```

Comment:
it would have been kinda nice to have these changes in a separate PR but I understand that's not quite how things evolved

Reply:
This one will likely be blocked, so I can look into extracting the relevant bits to a separate PR